### PR TITLE
docs: fix parameter links

### PR DESCRIPTION
**Pull request by njzjz**
It was forgotten in https://github.com/deepmodeling/deepmd-kit/pull/2549

## Summary by CodeRabbit

- **New Features**
	- Introduced the DPA-1 model with an attention mechanism for molecular simulation.
	- Added the `se_a_mask` descriptor for DP/MM simulations with dynamic atom counts.
	- Expanded support for multi-task fine-tuning in PyTorch.

- **Documentation Enhancements**
	- Improved clarity and detail in various documents, including model compression, DPLR model training, and fine-tuning processes.
	- Updated references to follow a standardized format, enhancing navigation and understanding.

- **Bug Fixes**
	- Corrected references and parameters across multiple documents to ensure accuracy in model configurations and training instructions.

### CODE REVIEW

File: doc/model/index.rst [hunk 1]
```diff
@@ -24,3 +24,4 @@ Model
    linear
    pairtab
    change-bias
+   precision
```
[the remark this answers is not in the document]
Comment (njzjz) [reply doc/model/index.rst:27]:
```suggestion
```

File: deepmd/utils/argcheck.py [hunk 2]
```diff
@@ -1387,14 +1387,16 @@
...
+    link_lf = make_link("loc_frame", "model[standard]/descriptor[loc_frame]")
+    link_se_e2_a = make_link("se_e2_a", "model[standard]/descriptor[se_e2_a]")
+    link_se_e2_r = make_link("se_e2_r", "model[standard]/descriptor[se_e2_r]")
+    link_se_e3 = make_link("se_e3", "model[standard]/descriptor[se_e3]")
+    link_se_a_tpe = make_link("se_a_tpe", "model[standard]/descriptor[se_a_tpe]")
+    link_hybrid = make_link("hybrid", "model[standard]/descriptor[hybrid]")
+    link_se_atten = make_link("se_atten", "model[standard]/descriptor[se_atten]")
+    link_se_atten_v2 = make_link(
+        "se_atten_v2", "model[standard]/descriptor[se_atten_v2]"
+    )
```
[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/utils/argcheck.py:1399]: I don't know why they are defined but unused. @wanghan-iapcm any idea here?

[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/utils/argcheck.py:1399]: Maybe

```suggestion
```

[the remark this answers is not in the document]
Comment (wanghan-iapcm) [reply deepmd/utils/argcheck.py:1399]: we may link the descriptors in the descriptor type doc `doc_descrpt_type`

**Review by wanghan-iapcm — APPROVED** [review]

File: deepmd/utils/argcheck.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (njzjz) [reply deepmd/utils/argcheck.py:1399]: I inserted links in https://github.com/deepmodeling/deepmd-kit/pull/4239/commits/178cfc6fa9029eb22ee78deb0a18729d8c62a2e3. But it looks like this should be done by dargs automatically...
